src/Hooks/CommandStartingListener.php

```diff
@@ -63,7 +63,7 @@ public function __invoke(CommandStarting $event): void
 
     private function registerJobHooks(): void
     {
-        $this->nightwatch->state->source = 'job';
+        $this->nightwatch->configureForJobs();
```

Comment:
Are we going to apply the same change to scheduled tasks in a separate PR?

Reply:
I would like to, yes.

I'm trying to keep the changes focused on each type as we go to keep review easier.